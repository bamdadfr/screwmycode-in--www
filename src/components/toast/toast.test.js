import React from 'react'
import renderer from 'react-test-renderer'
import ToastComponent from '@/components/toast/toast.component'

describe ('ToastComponent', () => {

    const Component = <ToastComponent/>
    const instance = renderer.create (Component).toJSON ()

    it ('should match snapshot', () => {

        expect (instance).toMatchSnapshot ()
    
    })

    it ('should have 1 children', () => {

        expect (instance.children).toHaveLength (1)
    
    })

})