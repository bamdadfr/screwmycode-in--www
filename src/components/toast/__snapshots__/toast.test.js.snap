// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastComponent should match snapshot 1`] = `
<div
  className="sc-19337ie-0 htDFPP"
>
  <div
    className="Toastify"
  />
</div>
`;
